pkg/api/v1/resource/helpers.go

```diff
@@ -227,6 +228,13 @@ func maxResourceList(list, newList v1.ResourceList) {
 	}
 }
 
+// EqualResourceList compares two ResourceList maps for equality.
+func EqualResourceList(list1, list2 v1.ResourceList) bool {
```

Comment:
Can you just use `apiequality.Semantic.DeepEqual` instead?

Reply:
fixed